src/index.js

```diff
@@ -35,6 +35,9 @@ module.exports = function(config) {
  * @returns {{get: get, save: save, all: all, allById: allById}}
  */
 function getStorageObj(client, namespace) {
+    function delete(id, cb) {
```

Comment:
I'm running this locally, turns out delete is a reserve word, could you change it to something else? Also you won't need a semi-colon after a function declaration. A little whitespace after the function would be nice too 😄  Otherwise things look great!